src/commit.md

```diff
@@ -0,0 +1,7 @@
+# `commit.md` - Rules for Git commit messages
```

Comment:
such a file might not want to go in `src` but rather in `res`?
not a big deal, that's just not really a source file :wink:

Reply:
I will move it to a `res` folder.

Reply:
looking forward to the change :yum: